src/Horarium/Builders/Fallback/FallbackJobBuilder.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Horarium.Builders.Parameterized;
+using Horarium.Fallbacks;
+using Horarium.Interfaces;
+
+namespace Horarium.Builders.Fallback
+{
+    internal class FallbackJobBuilder<TJob, TJobParam> : IFallbackJobBuilder where TJob : IJob<TJobParam>
```

Comment:
It looks very similar to `ParameterizedJobBuilder`, but there are problem with `JobBuilder` and `IJobBuilder`, which couples _real_ builder methods like `WithDelay`, `Next`, etc with `Schedule` method (which is not applicable here). Do you have any ideas how we could simplify it and reduce code duplication?

Reply:
Left `HorariumClient.Create()` method and `IParameterizedJobBuilder` untouched due to backward compatibility, marked them as Obsolete.
Created new `HorariumClient.Schedule()` method and `IJobSequenceBuilder` so now job building and scheduling are separated. Fallbacks configuring is only available using new `HorariumClient.Schedule()`.
And `FallbackJobBuilder` has been removed due to uselessness.